packages/perseus/src/widgets/explanation.tsx

```diff
@@ -71,85 +71,75 @@ class Explanation extends React.Component<Props, State> {
             : this.props.showPrompt;
 
-        let promptContainer: React.ReactNode;
-
-        // TODO(diedra): This isn't a valid href;
-        // change this to a button that looks like a link.
-        const href = "javascript:void(0)";
-        const onClick = this._onClick;
-
-        if (isMobile) {
-            promptContainer = (
-                <div className={css(styles.linkContainer)}>
-                    <a
-                        className={css(styles.mobileExplanationLink)}
-                        href={href}
-                        onClick={onClick}
-                        role="button"
-                        aria-expanded={this.state.expanded}
-                    >
-                        {promptText}
-                    </a>
-                    {this.state.expanded && (
-                        <svg className={css(styles.disclosureArrow)}>
-                            <polygon
-                                style={{fill: backgroundColor}}
-                                points={
-                                    `0,${arrowHeight} ` +
-                                    `${arrowWidth},${arrowHeight} ` +
-                                    `${arrowWidth / 2},0`
-                                }
-                            />
-                        </svg>
-                    )}
-                </div>
-            );
-        } else {
-            const viewStyling = isArticle
-                ? [styles.explanationLink, styles.articleLink]
-                : [styles.explanationLink, styles.exerciseLink];
-            promptContainer = (
-                <Clickable
-                    onClick={onClick}
-                    aria-expanded={this.state.expanded}
-                >
-                    {() => <View style={viewStyling}>{`[${promptText}]`}</View>}
-                </Clickable>
-            );
-        }
-
-        const expandedStyle = isMobile
-            ? styles.contentExpandedMobile
-            : styles.contentExpanded;
+        const caretIcon = this.state.expanded ? caretUp : caretDown;
+
+        const allowTransition = window.matchMedia(
+            "(prefers-reduced-motion: no-preference)",
+        ).matches;
+
+        // Special styling is needed to fit the button in a block of text without throwing off the line spacing.
+        // While the button is not normally included in a block of text, it needs to be able to accommodate such a case.
+        const buttonStyleOverrides = {
+            height: "22px",
+            marginLeft: "-2px",
+            padding: "0 2px",
+        };
+
+        const contentStyling = css(
+            styles.content,
+            this.state.expanded
+                ? styles.contentExpanded
+                : styles.contentCollapsed,
+            allowTransition &&
+                (this.state.expanded
+                    ? styles.transitionExpanded
+                    : styles.transitionCollapsed),
+        );
+
+        const contentTestId =
```

Comment:
**thought**: We typically haven't prevented test IDs in prod code. In my mind, it's not a big deal if we have test IDs in prod markup. It also removes the need for the test where we alter the `NODE_ENV` variable, which will be problematic when running tests in parallel.

Reply:
Removed.